functions/ts/kubeval/README.md

```diff
@@ -0,0 +1,32 @@
+# kubeval
+
+### Overview
+
+Validate KRM resources using [kubeval].
+
+### Synopsis
+
+`kubeval` allows you to validate KRM resources against their [json schemas].
+
+The json schemas can be provided via `schema_location` and
+`additional_schema_locations`. If neither `schema_location` nor
+`additional_schema_locations` is provided, we will convert the baked-in OpenAPI
+document to json schemas and use them.
+
+This function can be configured using a ConfigMap with the following keys, all
+of which are optional:
+
+- `schema_location`: The base URL used to download schemas.
+- `additional_schema_locations`: List of secondary base URLs used to download
```

Comment:
given `fn render` doesn't allow network access, do you want to expand on the usage of this.

Reply:
Updated.